### PR TITLE
Feat/search venues

### PR DESCRIPTION
**Pull request by ravinderreddy-p**
### What change does this PR introduce?

Search venues by names or post code. It supports fuzzy search (handles typos and spaces)

### Notes
run below command
- docker-compose -f local.yml up 

### Ticket

- [Trello Ticket](https://trello.com/c/ZmyRV9pu)

### Screenshots
image
image
image
image

### CODE REVIEW

**Comment by ravinderreddy-p** [comment]
Kindly review and merge with dev

File: sharespots/settings/base.py [hunk 1]
```diff
@@ -14,14 +14,14 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
 
-DEBUG = get_env('DJANGO_DEBUG', False)
+DEBUG = True
```
Comment (abuDarda97) [inline sharespots/settings/base.py:17]: @ravinderreddy-p do we need `DEBUG` to be set to true in our base setting file?

File: sharespots/settings/base.py [hunk 2]
```diff
@@ -14,14 +14,14 @@
...
 
 
 ############
 # Security #
 ############
 
-ALLOWED_HOSTS = get_list('ALLOWED_HOSTS', separator=',')
+ALLOWED_HOSTS = ['*']
```
Comment (abuDarda97) [inline sharespots/settings/base.py:24]: @ravinderreddy-p how does `ALLOWED_HOSTS = ['*']` work?

**Review by abu271 — COMMENTED** [review]
I pulled the branch locally, it ran fine and search feature is working well :+1:

**Review by abu271 — APPROVED** [review]

**Pull request merged**